IPs/RSKIPxxx.md

```diff
@@ -0,0 +1,262 @@
+
+The Rootstock bridge is designed to transfer fungible tokens, but not non-fungible elements. To transfer non-fungible elements, the bridge must segregate them, and avoid mixing them with other elements. The bridge is not prepared to do this now, and sending an UTXO containing an ordinal to the Bridge will cause the ordinal to be commingled with other UTXOs, and its ownership would be inadvertently transferred to a user performing a peg-out at a later time.
+
+# **Specification**
+
+This is a proposal for making Bitcoin ordinals and inscriptions available in Rootstock using the current Powpeg bridge. Once ordinals or inscriptions have representations in Rootstock,  they can be traded as ONFTs (Ordinal NFTs). At any moment, the owner of the ONFTs can redeem them in Bitcoin and transfer them to any Bitcoin address.
+
+Internally, the bridge does not know anything about NFTs or EIP-712. The bridge will only provide the service of the temporal custody of isolated UTXOs, what we’ll call parking. The bridge will provide one additional service: changing the owner of a parked UTXO. 
+
+Using external user-provided smart-contracts, those ordinals can be turned into EIP-712-compatible NFTs. To enable this, the community needs to define a standard URI for the lookup of ordinals in the Bitcoin blockchain, so that NFTs marketplaces can retrieve the associated inscription. That standard is out of scope of this document.
+
+To avoid attackers spamming the Bridge with ordinals, there are upfront and recurrent costs for parking ordinals within RSK. Ordinals whose annual fee has been unpaid will be assigned to the first user who pays its parking fees. In the future, a blinded auction process could be added.  
+ 
+The Bridge will provide these new methods:
+
+* **getFederationOrdinalsAddress**(): get the current address to transfer ordinals to Rootstock.
+* **registerBtcTransactionWithOrdinals**(): registers all the ordinals in a transaction. The ordinals are identified by being sent to the ordinals federation address.
+* **registerBtcTransactionWithOrdinalsDMFA**(): registers all ordinals in a transaction. Ordinals are identified by being sent to unique federation addresses mapped to a destination owner.
+* **getOrdinalOwner**(): Returns the RSK owner of an ordinal. 
+* **transferOrdinal**(): sets a new RSK owner for an ordinal. Only the owner can transfer ownership of  an ordinal.
+* **releaseOrdinal**(): transfers the ordinal back to a Bitcoin user of the Bitcoin blockchain. Only the owner can release an ordinal.
```

Comment:
Should we use the term `pegout` instead of release to keep it consistent with the 2wp?

Reply:
The current bridge method for peg-out is called releaseBtc(), so I thought "release" was more consistent.